test/e2e/specs/embedding.test.js

```diff
@@ -1,7 +1,15 @@
 /**
  * Internal dependencies
  */
-import { clickBlockAppender, newPost, isEmbedding, setUpResponseMocking, JSONResponse } from '../support/utils';
+import {
+	clickBlockAppender,
+	newPost,
+	isEmbedding,
+	setUpResponseMocking,
+	JSONResponse,
+	pressWithModifier,
+	META_KEY,
```

Comment:
You will have to rebase with `master` and update usage of `META_KEY`. It was removed in https://github.com/WordPress/gutenberg/commit/3b16e5fc7baceb64b0d1384e234a6077403dcb70 in favor of using `await pressWithModifier( 'primary', key );`